dnf-behave-tests/dnf/module/enable-errors.feature

```diff
@@ -22,32 +22,12 @@ Scenario: Fail to enable a different stream of an already enabled module (dnf)
         | nodejs    | enabled   | 8         |           |
     And stderr is
         """
+        Failed to resolve the transaction:
         The operation would result in switching of module 'nodejs' stream '8' to stream '10'
         Error: It is not possible to switch enabled streams of a module unless explicitly enabled via configuration option module_stream_switch.
         It is recommended to rather remove all installed content from the module, and reset the module using 'dnf module reset <module_name>' command. After you reset the module, you can install the other stream.
```

Comment:
I think the test fails because this line was removed based on: https://github.com/rpm-software-management/dnf5/pull/1285#discussion_r1514029591

Reply:
I removed the line in additional commit. Some tests still fail until https://github.com/rpm-software-management/ci-dnf-stack/pull/1483 is merged.